content/docs/querying/functions.md

```diff
@@ -162,6 +162,16 @@ more trends in the data is considered. Both `sf` and `tf` must be between 0 and
 
 `holt_winters` should only be used with gauges.
 
+## `idelta()`
+
+`idelta(v range-vector)`
+
+`idelta(v range-vector)` calculates the difference between the second-to-last
+and last value of each time series element in a range vector `v`, returning an
+instant vector with the given deltas and equivalent labels.
```

Comment:
Maybe "difference between the last two samples in the range vector" is easier to understand?